package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "firestore-meilisearch",
+    "version": "0.1.1",
```

Comment:
Why v0.1.1? The v0.1.0 is not even out

Reply:
Because I thought the first one was 0.1.1

Reply:
Good reason then 😄 
But no, the first one is `0.1.0`

Reply:
Thanks @curquiza I know it now 😄